File: Upcoder/Khối 11/Chương 6 Xâu/Copy Xâu/K11_CopyXau.cs
using System;
namespace Copyxau
{
    public class Program
    {
        static void Main(string[]args)
        {
            string a = Console.ReadLine();
            string [] input = Console.ReadLine().Split(' ');
            int n = int.Parse(input[0]);
            int m = int.Parse(input[1]);
    
            if (n < 0 || m >= a.Length || n > m)
            {
                Console.Write("gioi han khong dung");
            }
            else
            {
                Console.Write(a.Substring(n, m - n + 1));
            }
        }
    }
}

File: Upcoder/Khối 11/Chương 6 Xâu/Copy Xâu/K11_CopyXau.py
a = input()
n, m = map(int, input().split())
if n < 0 or m > len(a) or m < 0 or m >= len(a):
    print("gioi han khong dung", end="")
else:
    sub_string = a[slice(n, m + 1)]
    print(sub_string, end="")
